Formula/monkeysphere.rb

```diff
@@ -3,7 +3,8 @@ class Monkeysphere < Formula
   homepage "http://web.monkeysphere.info/"
   url "https://deb.debian.org/debian/pool/main/m/monkeysphere/monkeysphere_0.44.orig.tar.gz"
   sha256 "6ac6979fa1a4a0332cbea39e408b9f981452d092ff2b14ed3549be94918707aa"
-  revision 3
+  license "GPL-3.0-or-later"
```

Comment:
COPYING:

> Monkeysphere is distributed in the hope that it will be useful, but
> WITHOUT ANY WARRANTY; without even the implied warranty of
> MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
> General Public License for more details.
> 
> Monkeysphere Copyright 2007, and are all released under the GPL,
> version 3 or later.
>